microsoft-edge/developer/index.yml

```diff
@@ -114,34 +114,39 @@ landingContent:
 # Row 2 with 3 cards
 # =============================================================================
 # Card r2c1
-  - title: Progressive Web Apps
+  - title: Progressive Web Apps (PWAs)
```

Comment:
which .md files in pwa card in docset landing page?

Reply:
docset landing page > pwa card: 
* Deleted "Debug a pwa".
* Moved "Services workers" & "web app manifest" to Get Started section of card.
* Added "publish to store", in How-to section of card.